packages/core/workers/src/Profiler.js

```diff
@@ -40,16 +40,21 @@ export default class Profiler {
   session: Session;
 
   startProfiling() {
-    this.session = new Session();
-    this.session.connect();
+    try {
+      const inspector: any = require('inspector');
+      this.session = new inspector.Session();
+      this.session.connect();
 
-    return Promise.all([
-      this.sendCommand('Profiler.setSamplingInterval', {
-        interval: 100
-      }),
-      this.sendCommand('Profiler.enable'),
-      this.sendCommand('Profiler.start')
-    ]);
+      return Promise.all([
+        this.sendCommand('Profiler.setSamplingInterval', {
+          interval: 100
+        }),
+        this.sendCommand('Profiler.enable'),
+        this.sendCommand('Profiler.start')
+      ]);
+    } catch (err) {
+      throw err;
```

Comment:
why do you re-throw the same error here? Is the try..catch needed at all?

Reply:
-> https://github.com/parcel-bundler/parcel/pull/3850#discussion_r355049882

Reply:
Should we wrap in a diagnostic?

Reply:
Hey @devongovett, what do you mean by `diagnostic `?